tests/README.md

```diff
@@ -1,17 +1,35 @@
 # Chalk Testing
 
-This directory contains all the basic functionality tests for chalk.
-Tests are run via docker compose in a separate `tests` container which
-internally uses the `pytest` framework.
+This directory contains all the basic functionality tests for chalk. Tests are run via `make tests`, which uses `docker compose` to run the `tests` container which internally uses the `pytest` framework.
 
-## Requirements
+While `pytest` can be used to run the tests directly, it is recommended to use the makefile script instead, as that provides a more consistent developer experience.
+
+## Setup
+
+Before running tests, ensure that `docker` and `docker compose` are installed.
```

Comment:
Can we list dependencies and give an option for how to run this stuff NOT in docker?  Obviously some Linux-specifiic stuff would break, but:

a) The process does slow things down a lt
b) We probably should run tests suites natively on a Mac anyway.

Reply:
Have added instructions with several caveats.